test/models/amp/AmpSpec.scala

```diff
@@ -0,0 +1,200 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models.amp
+import java.time.{LocalDate, LocalDateTime, ZoneOffset}
+
+import config.ApplicationConfig
+import models.registrationprogress.{Completed, NotStarted, Section, Started}
+import play.api.libs.json.Json
+import uk.gov.hmrc.http.cache.client.CacheMap
+import utils.AmlsSpec
+import org.mockito.Mockito._
+import play.api.mvc.Call
+
+trait AmpValues {
+
+  val dateVal = LocalDateTime.now
+
+  val completeData = Json.obj(
+    "typeOfParticipant"     -> Seq("artGalleryOwner"),
+    "boughtOrSoldOverThreshold"     -> true,
+    "dateTransactionOverThreshold"  -> LocalDate.now,
+    "identifyLinkedTransactions"    -> true,
+    "percentageExpectedTurnover"    -> "fortyOneToSixty"
+  )
```

Comment:
just was wondering if this json is good representation of the data... is not percentageExpectedTurnover a number?

Reply:
Again - I wanted to store exactly as is currently stored in AMP Mongo. Makes passing the data back and forwards simpler.